lib/Url/UrlManager.php

```diff
@@ -359,6 +359,11 @@ public static function getRewriteUrl(\rex_extension_point $ep)
                     continue;
                 }
 
+                // Prüfen ob die clang ID mit dem Profil übereinstimmt
+                if ($clangId != $profile->getArticleClangId()) {
```

Comment:
`$profile->getArticleClangId()`

hier kann auch `0` zurückkommen. Das passiert, wenn man im Profil "alle Sprachen" auswählt. 
Kannst du diesen Fall mit überprüfen?

Reply:
Ist erledigt, so gut?

Reply:
Ich vermute das reicht nicht. 
Könntest du mir dein Beispiel geben, damit ich direkt das habe wo der Fehler passiert?

Reply:
Steht bereits im Beschrieb des Issues:
> Wenn das nicht geprüft wird, wird der Datensatz womöglich nicht gefunden, wenn ein Datensatz in verschiedenen Sprachen/Profilen aber im gleichen Namespace verfügbar ist.

Ich hatte das im Rahmen eines Projekts vor, aufs Datum guck, 3 Monaten. War aber eigentlich nichts besonderes, zwei Sprachen, Datensatz im selben Profil in verschiedenen Sprache verfügbar. Wenn ich mich in Sprache A auf der URL des Datensatzes befinde und dort die URL des selben Datensatzes nach Sprache B ausgeben wollte, funktionierte das nicht. URL war glaube ich einfach leer.

> Ich vermute das reicht nicht.
Bei mir hat's gefunzt. Vorher gab's ja gar keine Prüfung.

Reply:
Wie war der Datensatz angelegt. Man muss ja immer zwei Varianten prüfen.

A) Ein Datensatz enthält alle Übersetzungen (Variante mit `feld_clangId`) 
B) Pro Sprache existiert ein Datensatz in der Tabelle und wird via `clang_id`-Feld ermittelt.

Reply:
Es war Variante A)